elements/data-viz/src/data-viz.js

```diff
@@ -4,7 +4,6 @@
  */
 import { LitElement, html, css } from "lit";
 import "@haxtheweb/chartist-render/chartist-render.js";
-import "@haxtheweb/es-global-bridge/es-global-bridge.js";
```

Comment:
why did you delete this line?

Reply:
This was done by me, I added it and then deleted it. I misread the error, It was related to something in chartist-render.